examples/tritonserver/app/README.md

```diff
@@ -0,0 +1,12 @@
+# Triton server
+
+Triton server can be used to inference AI workloads using machine learning models. there are couple of referece AI workloads like fraud detection, Iris classification covered in the [repo](https://github.com/PDeXchange/ai-demos)
```

Comment:
Give official link also here.
And we can mention some of the pre built examples are given here, user can utilise them to try out the triton inference server.